app/models/schemas/workflow.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Schemas
+  ##
+  # Schema for Donut administrative metadata
```

Comment:
Looks like a copy/paste thing here, but can you update these docs for `Schemas::Workflow`?